packages/blade/src/components/Carousel/types.ts

```diff
@@ -1,14 +1,17 @@
 import type { Theme } from '~components/BladeProvider';
 import type { BoxProps } from '~components/Box';
 import type { Platform } from '~utils';
-import type { DotNotationColorStringToken } from '~utils/types';
+import type { DotNotationToken } from '~utils/lodashButBetter/get';
```

Comment:
Why is `DotNotationToken` in `lodashButButter/get`?

Reply:
It's a new improved type that I wrote, we will refactor out the older DotNotationColorStringToken things and replace it with this one later.